### DISCUSSION TITLE
Weather widget - Flyout shows only a solid background?

### DISCUSSION THREAD

**Discussion by doterik** [valinet/ExplorerPatcher#765] [Q&A] [answered]
I have trouble seeing the flyout on the weather widget, there's **only a solid background**.

The image is evidently there because the mouse pointer changes shape between an arrow and a hand (where you normally can click). If I click on the Wind button then the Temperature button changes to a hand. (I do this blindly.)

Probably this appeared in the latest version (22000.469.42.4), not quite sure. What I do know is that it worked perfectly in the past.

Don't know if there may be something with color settings but I haven't changed any of these in a long time.

Any tips on a setting that would cause the problem?

###### (Everything is restarted several times, mainly because of turning on/off the computer.)

**Comment by valinet** [comment, marked as answer]
Things may be broken in the latest pre-releases, I broke some things, fixed some of them in my local copy, still have to do some work but did not have the time to finish. I will improve it in the coming builds, it’s a pre-release, I can’t always work on this at exactly the right time, so to say. I ought to fix it, and will soon, I use this daily as well, it has to work great.

**Reply by doterik** [reply]
Great, no worries... just fine to know it's not something with my settings.
Thanks 👍🏼

**Comment by doterik** [comment]
With the latest version, this now works again... 🎉

![image](https://user-images.githubusercontent.com/4037602/152807390-d99bcbb5-d6f7-4be6-9235-779d16a2ee8a.png)

Thanks for the update. 👏🏼 👍🏼

**Reply by valinet** [reply]
You’re welcome, I am glad that it is fine now and works as intended.
